### PR TITLE
fix: [CDS-59602]: added selective stage exec on detail page

### PR DESCRIPTION
**Pull request by sahithikolichala**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by sahithikolichala** [comment]
retrigger jest

File: src/modules/72-triggers/pages/triggers/TriggersDetailPage.tsx [hunk 1]
```diff
@@ -155,27 +155,42 @@ const getDetailsContent = ({
...
+    {
+      label: getString('triggers.pipelineExecutionInput'),
+      value: !isEmpty(pipelineInputSet) ? <pre>{pipelineInputSet}</pre> : undefined,
+      type: ContentType.CUSTOM
+    }
+  ]
+  if (selectiveStageFF) {
+    arr.push({
+      label: getString('triggers.selectStagesToExecute'),
+      value: stagesToExecute?.length ? <p>{stagesToExecute}</p> : <p>All Stages</p>
```
Comment (pankajsingh-harness) [inline src/modules/72-triggers/pages/triggers/TriggersDetailPage.tsx:189]: This should comes from yaml

**Comment by sahithikolichala** [comment]
retrigger cypress-pipeline

**Comment by sahithikolichala** [comment]
retrigger titlecheck

**Comment by sahithikolichala** [comment]
retrigger titlecheck

**Review by pankajsingh-harness — APPROVED** [review]

**Review by cjlee01 — APPROVED** [review]

**Pull request merged**